pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = -s --ignore=app/controllers/canvasapi-develop
```

Comment:
`pytest` will blow up trying to read the canvasapi-develop stuff, so here we tell it to ignore it. We also pass `-s` to allow us to see print statements in tests.